r/factset.protobuf.stach.v2/DESCRIPTION

```diff
@@ -1,9 +1,11 @@
 Package: factset.protobuf.stach.v2
 Type: Package
+Version: 1.0.6
 Title: 'FactSet' 'STACH V2' Library
-Version: 1.0.3
-Author: analytics-reporting
-Maintainer: Charlie Mathis<chmathis@factset.com>
+Authors@R: c(person("Charlie Mathis", "Developer", role = c("aut", "cre"),
+                     email = "chmathis@factset.com"))
+Author: Charlie Mathis Developer [aut, cre]
+Maintainer: Charlie Mathis Developer <chmathis@factset.com>
```

Comment:
I think that we can remove these two lines as the same information is being conveyed in `line 5`. Can you please confirm it?

Reply:
Updated